tests/src/stronk_tests.cpp

```diff
@@ -34,6 +35,146 @@ TEST(unwrap, when_unwrapping_it_works_with_its_own_argument)  // NOLINT
     EXPECT_EQ(b.unwrap<a_float_test_type>(), 42.1F);
 }
 
+template<an_int_test_type Val>
+struct type_which_requires_stronk_none_type_template_param
+{
+    constexpr static an_int_test_type value = Val;
+};
+
+TEST(none_type_template_parameter, stronk_types_can_be_used_as_none_type_template_parameters)  // NOLINT
```

Comment:
What does this NOLINT silence? (Consider adding the check to the comment, if it is not just a leftover)

Reply:
Some thing that the great macro creates 🙃

Reply:
I'll see what it is